DOT-1444 language sync to wiki

edwh opened the pull request:
As discussed, a cheap and cheerful command to sync the languages.

ngm commented on app/Providers/ScheduleServiceProvider.php:
```diff
@@ -0,0 +1,21 @@
+use Illuminate\Support\ServiceProvider;
+use Illuminate\Console\Scheduling\Schedule;
+
+class ScheduleServiceProvider extends ServiceProvider
```
Is this auto-registered somehow?

edwh replied:
No, it's registerd in app.php around line 173.

ngm replied:
Ah right, yes, missed that somehow :+1:

ngm reviewed (approved):
Looks good to me - just wondering how we ScheduleServiceProvider is registered (I don't remember how this works, so perhaps it's just automagically registered by virtue of being in the right folder)

Also - does it need a cron job to cause the schedule to run every 5 minutes, or Laravel schedules it itself?

edwh commented:
> Also - does it need a cron job to cause the schedule to run every 5 minutes, or Laravel schedules it itself?

It does need a cron job.  When I added a scheduled task to the Repair Directory I added this on production:

`* * * * * cd /srv/users/serverpilot/apps/repairdirectory && php artisan schedule:run >> /tmp/laravel_schedule.out 2>&1`

So we'd do something similar, and tweak the output file name.
